components/brave_rewards/browser/rewards_service_browsertest.cc

```diff
@@ -2126,7 +2126,7 @@ IN_PROC_BROWSER_TEST_F(BraveRewardsBrowserTest,
   rewards_service_->AddObserver(this);
 
   // Enable Rewards
-  EnableRewardsViaCode();
+  EnableRewards();
```

Comment:
This test using `EnableRewardsViaCode` would not show publisher info on panel. Reverted to `EnableRewards`